linux-build.sh

```diff
@@ -2,7 +2,33 @@
 
 set -euxo pipefail
 
+ARCH=""
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --GOARCH)
+    ARCH="$2"
+    shift # past argument
+    shift # past value
+    ;;
+    --default)
+    DEFAULT=YES
+    shift # past argument
+    ;;
+    *)    # unknown option
+    POSITIONAL+=("$1") # save it in an array for later
+    shift # past argument
+    ;;
+esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+
 VERSION_TAG="$(git rev-parse HEAD)"
 VERSION_LINK_FLAG="main.gitCommit=${VERSION_TAG}"
 
-CGO_ENABLED=0 GOOS=linux go build -a -ldflags "-extldflags '-static' -X ${VERSION_LINK_FLAG}" .
+CGO_ENABLED=0 GOOS=linux GOARCH=$ARCH go build -a -ldflags "-extldflags '-static' -X ${VERSION_LINK_FLAG}" .
```

Comment:
I don't think these changes are required. If `GOARCH` is declared as an `ARG` in the Dockerfile (with default `amd64`) then it will be passed to this script as an environment variable and picked up by the go compiler automatically.

Reply:
Yes yes, you were quicker than my commit: This is not needed.